ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/events/CreateTableEvent.java

```diff
@@ -19,23 +19,27 @@
 
 package org.apache.hadoop.hive.ql.security.authorization.plugin.metastore.events;
 
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Put them back in L#36. L#37 is not used, so we can remove it.

Reply:
In L#37, this import  org.apache.hadoop.hive.ql.security.authorization.plugin.metastore.HiveMetaStoreAuthzInfo is used in  L#59. 
The ArrayList and List are both used in L#65.